.ruby-version

```diff
@@ -1 +1 @@
-2.2.2
```

Comment:
This is only used in local development and it was out of date compared to the Gemfile which had 2.3.5 so now they are all in sync at 2.4.0
We should make a new issue to upgrade to ruby 2.5 or 2.6 when that comes out also- this is a continuing journey :) https://www.ruby-lang.org/en/downloads/releases/